recipes/LibriSpeech/ASR/CTC/hparams/train_hf_whisper_encoder.yaml

```diff
@@ -71,7 +71,6 @@ output_neurons: 29  # BPE size, index(blank/eos/bos) = 0
 
 # Decoding parameters
 blank_index: 0
-use_language_modelling: False
 # ngram_lm_path: !PLACEHOLDER
```

Comment:
this artefact can be removed as the training .py file doesn't have any beam search decoding fn.